cm_service_restart.py

```diff
@@ -3,7 +3,7 @@
 import argparse
 import os
 import sys
-
+import smtplib
```

Comment:
i dont see this being used anywhere..? i assume you're planning to send update emails